src/features/user/UserProfile/screens/IndividualProfile.tsx

```diff
@@ -113,11 +115,11 @@ export default function IndividualProfile({ userprofile, changeForceReload,
         autoHideDuration={4000}
         onClose={handleTextCopiedSnackbarClose}
       >
-        <MuiAlert 
-        elevation={6} 
-        variant="filled"
-        onClose={handleTextCopiedSnackbarClose} 
-        severity="success"
+        <MuiAlert
+          elevation={6}
+          variant="filled"
+          onClose={handleTextCopiedSnackbarClose}
+          severity="success"
         >
           Text Copied to Clipboard!
```

Comment:
@captainamiedi should this be translated?
@harshvitra do we still plan to use this alerts?

Reply:
These has been replaced with the share functionality.

Reply:
@captainamiedi Can you clean up the code?

Reply:
ok